lite/test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+if(POLICY CMP0079)
+    cmake_policy(SET CMP0079 NEW)
+endif()
+
+include_directories(
+    ${CMAKE_SOURCE_DIR}/include/libxmp-lite
+    ${CMAKE_SOURCE_DIR}/src)
+
+add_executable(libxmp-test test.c ../src/md5.c)
+set_property(TARGET libxmp-test PROPERTY C_STANDARD 90)
+if(WIN32) # Make sure unit test will find its DLLs while running
+    set_property(TARGET libxmp-test PROPERTY RUNTIME_OUTPUT_DIRECTORY ${libxmp_BINARY_DIR})
+endif()
+
+target_compile_definitions(libxmp-test PRIVATE -DLIBXMP_NO_DEPACKERS)
```

Comment:
This shouldn't be needed any more.

Reply:
Forgot to remove it, thanks for the pointing!